packages/devextreme/js/localization/messages/ar.json

```
@@ -699,6 +699,8 @@
         "dxGantt-showResources": "Show Resources",
         "dxGantt-showDependencies": "Show Dependencies",
         "dxGantt-dialogStartDateValidation": "Start date must be after {0}",
-        "dxGantt-dialogEndDateValidation": "End date must be after {0}"
+        "dxGantt-dialogEndDateValidation": "End date must be after {0}",
+
+        "dxGallery-galleryItemName": "Gallery item"
```

Why we duplicate `gallery`?
Seems like there can not be other item except gallery item.
```suggestion
        "dxGallery-itemName": "Gallery item"
```

---

fixed